janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLConfigOptions.java

```diff
@@ -65,6 +65,18 @@
                 "and always use LOCAL_QUORUM instead",
             ConfigOption.Type.MASKABLE, false);
 
+    ConfigOption<Boolean> USE_EXTERNAL_LOCKING = new ConfigOption<>(
+            CQL_NS,
+            "use-external-locking",
+            "True to prevent JanusGraph from using its own locking mechanism because an external locking " +
```

Comment:
I took another glance and have some recommendations for the config option description. 
```
                "True to prevent JanusGraph from using its own locking mechanism. Setting this to true eliminates " +
                "redundant checks when using an external locking mechanism outside of JanusGraph. Be aware that " + 
                "when use-external-locking is set to true, that failure to employ a locking algorithm which locks " +
                "all columns that participate in a transaction upfront and unlocks them when the transaction ends, " +
                "will result in a 'read uncommitted' transaction isolation level guarantee. If set to true without " + 
                "an appropriate external locking mechanism in place side effects such as " +
                "dirty/non-repeatable/phantom reads should be expected.",
```

Reply:
Fixed.